pkg/kubectl/cmd/util/factory_object_mapping.go

```diff
@@ -330,6 +390,16 @@ func (f *ring1Factory) AttachablePodForObject(object runtime.Object) (*api.Pod,
 		sortBy := func(pods []*v1.Pod) sort.Interface { return sort.Reverse(controller.ActivePods(pods)) }
 		pod, _, err := GetFirstPod(clientset.Core(), t.Namespace, selector, 1*time.Minute, sortBy)
 		return pod, err
+
+	case *extensions.ReplicaSet:
```

Comment:
There is an open PR for attach, mind leaving that aside?
https://github.com/kubernetes/kubernetes/pull/40365